src/BreakpointViewer.h

```diff
@@ -11,6 +11,16 @@
 class QPaintEvent;
 class Breakpoints;
 
+struct AddressRange {
+   int start;   
+   int end; 
```

Comment:
It might be a good idea to document:
* That `end` is inclusive. Because often in c++, or programming in general, the end-point of a range is exclusive, like in a mathematical half-open range.
* That `end < start` means a single address rather than a range. Depending on how often this knowledge is repeated, it might even be a good idea to extract this into a method: `bool isSingleAddress() const { return end < start; }`.

Alternatively, and maybe more logical, a single address could be represented as `start == end`, with an invariant `start <= end`. (IMHO that eliminates the need for a method like `isStartAddress()`, it's now obvious when a range contains only a single address.)